cdap-ui/app/cdap/components/PipelineConfigurations/Store/index.js

```diff
@@ -129,8 +132,8 @@ const getCustomConfigFromProperties = (properties = {}) => {
   return customConfig;
 };
 
-const getCustomConfigForDisplay = (properties, engine) => {
-  let currentCustomConfig = getCustomConfigFromProperties(properties);
+const getCustomConfigForDisplay = (properties, engine, isBatch) => {
+  let currentCustomConfig = getCustomConfigFromProperties(properties, engine, isBatch);
```

Comment:
Here aren't you passing in 3 parameters to `getCustomConfigFromProperties` even though the function expects 2?